.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: build
+
+on: [push]
+
+jobs:
```

Comment:
We can't officially support mac at this time, but it is good to have this build running to ensure that we don't do anything that would prevent us from supporting it in the future.

Could you please add a comment saying that there are no official mac releases, and mac is not being tested?

Would you mind adding a comment here to that effect.